src/ExpressionParser.php

```diff
@@ -750,15 +750,15 @@ private function getTest(int $line): TwigTest
         $stream = $this->parser->getStream();
         $name = $stream->expect(Token::NAME_TYPE)->getValue();
 
-        if (!$test = $this->env->getTest($name)) {
-            if ($stream->test(Token::NAME_TYPE)) {
-                // try 2-words tests
-                $name = $name.' '.$this->parser->getCurrentToken()->getValue();
+        if ($stream->test(Token::NAME_TYPE)) {
+            // try 2-words tests
+            $name = $name.' '.$this->parser->getCurrentToken()->getValue();
 
-                if ($test = $this->env->getTest($name)) {
-                    $stream->next();
-                }
+            if ($test = $this->env->getTest($name)) {
```

Comment:
shouldn't this have a `else` to use the 1-word test ?

Reply:
Having something after the test is not valid anyway. So, I prefer to have an exception saying `Unknown "empty element" test.` instead of an exception saying `Unexpected token "name" of value "element" ("end of statement block" expected).`